ddtrace/propagation/http.py

```diff
@@ -414,7 +417,9 @@ def _inject(span_context, headers):
             log.debug("tried to inject invalid context %r", span_context)
             return
 
-        single_header = "{}-{}".format(_dd_id_to_b3_id(span_context.trace_id), _dd_id_to_b3_id(span_context.span_id))
+        single_header = "{}-{}".format(
+            _dd_id_to_hexadec_id(span_context.trace_id), _dd_id_to_hexadec_id(span_context.span_id)
+        )
```

Comment:
since we are touching these lines
```suggestion
        single_header = "-".join((
            _dd_id_to_hexadec_id(span_context.trace_id), _dd_id_to_hexadec_id(span_context.span_id)
        ))
```
should have slightly better performance